yamllint_junit_bootstrap/bootstrap.py

```diff
@@ -39,31 +50,15 @@ def main():
         usage="%(prog)s [options] input",
     )
 
-    parser.add_argument('input', nargs='?', type=argparse.FileType('r'), default=("" if sys.stdin.isatty() else sys.stdin))
```

Comment:
All of this functionality is theoretically completely handled by default.
If this were to read:
```
   parser.add_argument('input', type=argparse.FileType('r'))
```
argparser handles checking the file and handles opening it and, if a user provides `-` as the argument, it will be read from stdin. is_pipe and such handling could then be removed and simply this a lot.
Though this would mean a breaking change (_an_ input argument would always be required)